cypress/integration/seededFlows/articleFlows/postModerationTools.spec.js

```diff
@@ -151,6 +151,52 @@ describe('Moderation Tools for Posts', () => {
         });
       });
     });
+
+    context('when unpublishing all posts', () => {
+      beforeEach(() => {
+        cy.get('@moderatorUser').then((user) => {
+          cy.loginAndVisit(user, '/series_user/series-test-article-slug');
+          cy.findByRole('heading', { level: 1, name: 'Series test article' });
+          cy.findByRole('button', { name: 'Moderation' }).click();
+        });
+      });
+
+      it('shows Unpublish All Posts button', () => {
+        cy.getIframeBody('[title="Moderation panel actions"]').within(() => {
+          cy.findByRole('button', { name: 'Open moderator actions' })
+            .as('moderatingActionsButton')
+            .pipe(click)
+            .should('have.attr', 'aria-expanded', 'true');
+
+          cy.findByRole('button', {
+            name: /Unpublish all posts of series_user/i,
+          }).should('exist');
+        });
+      });
```

Comment:
(not blocking) - I think we can get away without this spec, as the next one relies on finding the same button and should produce an understandable error if it doesn't

Reply:
works for me!